polly/plugins/maya/publish/submit_deadline.py

```diff
@@ -25,7 +25,8 @@ def process(self, instance):
         from avalon import api
         from avalon.vendor import requests
 
-        assert api.Session["AVALON_DEADLINE"], "Requires AVALON_DEADLINE"
+        deadline = api.Session.get("AVALON_DEADLINE", None)
```

Comment:
Just hit me, when we do the assertion we don't have to put it in a variable anymore.
We can directly do `"{}/api/jobs?JobID=%s".format(api.Session["AVALON_DEADLINE"])`, less code ;)